### DISCUSSION TITLE
Selling left-over kits

### DISCUSSION THREAD

**Discussion by ezcGman** [sanni/cartreader#810] [Hardware]
Hey there,

as this is a "sensible" topic as some hate offering this in a forum like this and others welcome it, I wanted to check how Sanni / admins feel about it:

I currently still have two complete / ready-to-build rev5 kits left over which I would be willing to resell. As you always get min 5 PCBs when ordering from any PCB manufacturer, I tend to also always go the extra mile and buy components to have five complete kits. This lowers the costs per kit dramatically, so it's basically a win-win when I sell the left over kits I don't need, as the overall price for me for a kit is pretty low (bit more than 50€) and also for four other people ;)
I build a lot of LED controllers and other DIY boards, so I'm used to buy components for all PCBs I order, make kits and sell the left over kits. I often do this.

But I wanted to ask if it would be fine if I (or others) would offer these kits here, as everybody has at least left over PCBs when building an OSCR. May be a nice new section here ("Marketplace") or in a potential Discord server?

Sorry if I crossed a line or stepped on somebodies foot with this!

Greetings,

Andy!

**Comment by nsx0r** [comment]
Hi, there is a trading area in Discord: https://discord.gg/2SJUKGyt

**Comment by ezcGman** [comment]
Oh, I looked for a Discord on the GH start page but couldn't find one! Alright, I'll gonna post there then :) Thx!
